### ISSUE TITLE
[PlotTwistNoFansub] Connector not working

### ISSUE THREAD

**Issue by ralvarador** [manga-download/hakuneko#7150]
### Is there an existing issue for this connector?

- [X] I have searched the existing issues

### Did you read the troubleshooting guide ?

- [X] I read it but nothing seems related

### Is the website of the connector working properly ?

- [X] Yes i'm able to see the manga within my standard browser

### Make sure to try Nightly Build

- [X] Yes nightly build didn't fix the issue

### What kind of issue are you encountering

Downloads aren't working

### Describe the bug

Website changed their domain from www[.]plot-twistnf-scans[.]com to www[.]plotfansub[.]com, so modified the .mjs connector file reflecting the change of domain, and the download of the chapters worked again.

They did other changes that affect the line: this.paths = [ '/proyectos-finalizados-new/', '/proyectos-activosss/' ];
"proyectos-finalizados-new" = still remains in the previous domain. "proyectos-activosss" = is present in the new domain but modified to "proyectos-activos".

Recently the "path" of the titles was modified from "/plot/manga/<title_of_manga>" to "/plotwist/manga/<title_of_manga>", so I modified the hakuneko.bookmarks to reflect the change of the path.
"key": {
            "connector": "plottwistnofansub",
            "manga": "/plot/manga/kami-samaa-ga-machigaeruu/" => "manga": "/plotwist/manga/kami-samaa-ga-machigaeruu/"
        }
With this last change, Hakuneko was not able of "detect" the "title" chapters available, despite of get the right URL and its content.
![imagen](https://github.com/manga-download/hakuneko/assets/30580377/b2eb5027-a2c1-4d88-9ed7-2f7a2ed157af)

Hakuneko details:
Version: 6.1.7 - d81c39
Architecture: x64
Note: Portable mode active (-p parameter passed by command line)

### To Reproduce

1. Name of connector: PlotTwistNoFansub
2. Name of manga (if connector update is working): Pleasee Put These On, Takaminee
3. Name of chapter (if manga update is working): Capítulo 46.60: EL MEJOR VESTIDO DE LA PRESIDENTA
4. Copy any relevant error message that appears in a popup or (advanced) within the console (press F12): None.

**Comment by MikeZeDev** [comment]
No, the url hasnt changed.  There is no URL change at all .
The one we support (https://www.plot-twistnf-scans.com/proyectos-finalizados-new/) is for "finished projects", 

![image](https://github.com/manga-download/hakuneko/assets/10671518/ee5ad05d-7569-426d-83ae-6137c8d6a650)

and is working

![image](https://github.com/manga-download/hakuneko/assets/10671518/dbb1df24-baea-449f-a57e-76162ebf5cc0)

the one you want (https://www.plotfansub.com/proyectos-activos/) is for "current projects"

![image](https://github.com/manga-download/hakuneko/assets/10671518/1b2d893c-927e-4d38-8f6c-4b67c4ca9234)


TLDR : you want a different website, make a "website suggestion" issue.

**Comment by ralvarador** [comment]
Noted... The scanlation group - Finished Project website is the supported, and the one for the "current projects" (www[.]plotfansub[.]com) must be reported as new "website suggestion", right?
